### PR TITLE
Fixes #7.

### LINKED ISSUE #7: Documentation for .swift-version file formats
Missing in readme. I'd add it but I'm about to board a plane.

### PR DESCRIPTION
**Pull request by ashfurrow**
:boom:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,13 +36,15 @@ Fall back to the Swift defined by your `xcode-select` configuration:
...
 ```bash
-$ swift --version
-Swift version 1.1 (swift-600.0.57.4)
-Target: x86_64-apple-darwin14.3.0
 $ echo 1.2 >.swift-version
+```
+
+You can verify it's working using the following. 
+
+```base
```
Comment (neonichu) [inline README.md]: This should be `bash`

**Comment by neonichu** [comment]
:sparkling_heart:

File: README.md (hunk 1 above)
Comment (ashfurrow) [inline README.md]: UGH I'll have to actually clone to fix this. Hold on.

**Comment by ashfurrow** [comment]
:boom: again

**Pull request merged**